src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -212,13 +200,13 @@ public Base(final Request req) throws IOException {
                 throw new HttpException(
                     HttpURLConnection.HTTP_BAD_REQUEST,
                     String.format(
-                        // @checkstyle LineLength (1 line)
-                        "boundary is not specified in Content-Type header: \"%s\"",
+                        "boundary is not specified in Content-Type header:",
```

Comment:
@2686747 same here, don't change this, it will cause the message to be broken. Just leave the suppressed warning in place.